site/en/install/ubuntu.md

```diff
@@ -39,9 +39,9 @@ Bazel comes with two completion scripts. After installing Bazel, you can:
 
 ```posix-terminal
 sudo apt install apt-transport-https curl gnupg
-curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
-sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
-echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
+sudo mv bazel-archive-keyring.gpg /usr/share/keyrings
```

Comment:
Is it possible that this directory doesn't exist yet? If so, a `mkdir -p` could help.

Reply:
No, the directory `/usr/share/keyrings` is always there and contains gpg keys for Ubuntu repositories.